docs/usage-examples.txt

```diff
@@ -0,0 +1,93 @@
+.. _laravel-usage-examples:
+
+==============
+Usage Examples
+==============
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: set up, runnable, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. TODO: add usage examples to TOC as they're made
+   .. toctree::
+
+
+Overview
+--------
+
+Usage examples show runnable code examples to demonstrate frequently used MongoDB
+operations. Each usage example includes the following:
+
+- Description of the MongoDB operation
+- Code example that runs the MongoDB operation, which you can paste into your controller file
+- Code example that specifies a view, which you can paste into your view file
+- Output returned by the view
+
+How to Use the Usage Examples
+-----------------------------
+
+The usage examples are designed to run operations on a MongoDB deployment that contains
+the Atlas sample datasets. When you run the example code without the sample data, the output
+might not match. 
+
+Follow this tutorial to set up your MongoDB deployment with the sample data and run the
+example code in your development environment. Before performing the following actions,
+ensure that you create an Atlas account and deploy a cluster. For information about setting
+up an account and a cluster, see the :atlas:`Get Started with Atlas Guide
+</getting-started/#atlas-getting-started>`.
```

Comment:
Issue:

1) I think the description of the procedure/steps is currently inaccurate. When reading the first sentence "Follow this tutorial to set up your MongoDB deployment with sample data and run the sample code...", I expected the tutorial to explain that part. However, the procedure/steps includes the prerequisite of the quick start guide which includes setup of the MongoDB deployment with sample data and how to add and run the code. 

2) I think the step in the tutorial about running the usage example code is inaccurate since it omits information about viewing it in the browser.

Suggestion:

I think keeping the description of the steps and the procedure/steps aligned would be helpful. I also think this page should delegate some of the explanation of how to run them to the specific usage example since the procedure might differ depending on the operation.

E.g.

"You can run the usage examples from Laravel web application and MongoDB Atlas cluster that you set up in the :ref:`laravel-quick-start` guide. Ensure that your application and cluster meet the following requirements:
...

To run the example controller code and view the results, follow the instructions on the usage example page.
"

Let me know if you need any help with filling out the details.

Reply:
Thanks for this detailed comment; I think I made all the changes you requested, let me know if it looks alright